application/src/main/java/org/opentripplanner/transit/model/basic/Money.java

```diff
@@ -147,6 +147,11 @@ public Money half() {
     return new Money(currency, IntUtils.round(amount / 2f));
   }
 
+  public Money roundDownToNearestFiveCents() {
```

Comment:
I think you know the drill by now: this one needs Javadoc and a test.

Reply:
Oops! I should know better. Test added now